Detections/MultipleDataSources/TrafficToKnownBadIPs.yaml

```diff
@@ -18,35 +18,36 @@ requiredDataConnectors:
     - CommonSecurityLog
```

Comment:
Can CommonSecurityLog have a connectoId?
if yesm, please move CommonSecurityLog  to another item and add the connector id